pkg/inject/sidecar_builder.go

```diff
@@ -325,6 +331,19 @@ func envVar(envName, envVal string) corev1.EnvVar {
 	}
 }
 
+// refPodUid is to use the k8s downward API and render pod uid
+func refPodUid(envName string) corev1.EnvVar {
```

Comment:
Could you elaborate little more in comments about the usage ?

Reply:
Yeah, will add more comments about it